plans/all.fmf

```diff
@@ -1,5 +1,14 @@
 discover:
     how: fmf
+
+# counteract https://issues.redhat.com/browse/TFT-2564
```

Comment:
Note: With the second commit of re-disabling c10s tests this commit isn't strictly necessary any more. But I'd still like to keep it: It doesn't hurt, it was very non-trivial to figure out, and once the iptables bug gets fixed we'll want to re-enable c10s tests.